modules/tinymce/src/themes/silver/main/ts/ui/menus/item/structure/ItemSlices.ts

```diff
@@ -11,17 +11,12 @@ import { Optional } from '@ephox/katamari';
 
 import I18n from 'tinymce/core/api/util/I18n';
 
-import { get as getIcon, IconProvider } from '../../../icons/Icons';
+import { get as getIcon, IconProvider, render as renderIconElement } from '../../../icons/Icons';
```

Comment:
Can we also change this to `* as Icons` please? That'll also avoid having to come up with different names like we are now.